src/modules/admin/components/AdminDashboard/AdminDashboard.tsx

```diff
@@ -152,11 +161,12 @@ const AdminDashboard = ({
     [colonyName],
   );
 
-  const { error: colonyError, data: colony } = useDataSubscriber(
-    colonySubscriber,
-    [colonyAddress],
-    [colonyAddress],
-  );
+  const {
+    data: { colony } = {},
```

Comment:
Please do not use defaults here, just do a check whether `data` is defined and if not, show a loading spinner